src/components/Tag/Tag.module.css

```diff
@@ -0,0 +1,99 @@
+@import '../../design-tokens/mixins.css';
+
+/*------------------------------------*\
+    #TAGS
+\*------------------------------------*/
+
+/**
+ * The rounded rectangle container that houses information.
+ * 1) Grabs colors from whichever variant is selected.
+ * 2) Rounds the corners of the tag.
+ */
+.tag {
+  width: max-content;
+  max-width: 20rem;
+  padding-top: var(--eds-size-half);
+  padding-bottom: var(--eds-size-half);
+  padding-left: var(--eds-size-1-and-half);
+  padding-right: var(--eds-size-1-and-half);
+
+  display: inline-flex;
+  align-items: center;
+
+  border: var(--tag-secondary-color) solid var(--eds-theme-border-width); /* 1 */
+  border-radius: 9999px; /* 2 */
+
+  /** 
+   * TODO: (dcwither) when adding SvgIcon to EDS, include this in tailwind font 
+   * size macro.
+   */
+  --svg-icon-size-default: 1.25rem;
```

Comment:
Discussing in https://github.com/chanzuckerberg/edu-design-system/pull/941#discussion_r843047983, resolving

Reply:
I think it is relevant but now it's called `--icon-size` and maybe we should be passing it in via the `size` prop instead of setting the css variable directly?

https://github.com/chanzuckerberg/edu-design-system/pull/940/files

Reply:
I will change it to `--icon-size` to match, but don't think allowing icon variations in the tag is necessary since there are no size variations for the text and the `Icon` is being passed as a prop and can be styled indepedently

Reply:
Changing to `--icon-size-default` as per https://github.com/chanzuckerberg/edu-design-system/pull/941#discussion_r843303384